published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,209 @@
+  - [@noffle](//github.com/noffle) also made progress toward getting `go-ipfs` compatible with vanilla `go get` installation. Hope to have more tangible results to show by end of the month.
+  - [@RichardLitt](//github.com/RichardLitt) made a lot of documentation fixes to the CLI while working on the IPFS HTTP API.
+  - [@lgierth](//github.com/lgierth) improved the automated [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs, which is now named `ipfs/go-ipfs`. Give it a try: `docker run -i --net=host ipfs/go-ipfs`. The new image is built automatically for every commit, and has automatic tags for the coming releases. It's also a few MB smaller, and doesn't require mounting a volume for the IPFS repository anymore. Instead, if there's no volume mounted, it'll generate an ephemeral identity and configuration which will be lost when the container exits. This is ideal for testing or just trying something out real quick. Images for go-ipfs 0.3.x are not provided, because the Dockerfile shipped with these version isn't suitable for Docker Hub automatic builds.
+  - [@chriscool](//github.com/chriscool) rewrote [the build documentation for Windows](//github.com/ipfs/go-ipfs/blob/master/docs/windows.md), which enabled [@slothbag](//github.com/slothbag) to get Windows builds working again with minimal fuss.
+  - [@lgierth](//github.com/lgierth) posted a writeup on the [migration from 0.3.x to 0.4.0](https://ipfs.io/blog/9-v04x-migration/).
+* [**specs**](//github.com/ipfs/specs/pull/37) The IPLD spec was merged, after four months. This reflected a ton of work from [@mildred](//github.com/mildred), [@findkiko](//github.com/findkiko), and [@jbenet](//github.com/jbenet). The [IPLD spec](https://github.com/ipfs/specs/blob/master/merkledag/ipld.md), the "thin-waist" merkle dag format, defines merkle-links, -dags, and -paths, as well as the IPLD Data model and formats. In short: JSON documents with named merkle-links that can be traversed.
+* [**distributions**](//github.com/ipfs/distributions)
+  - [dist.ipfs.io](http://dist.ipfs.io/) _almost_ has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to [@dignifiedquire](//github.com/dignifiedquire). This will be coming soon.
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
+	- [@RichardLitt](//github.com/RichardLitt) spruced up the [gx README](https://github.com/whyrusleeping/gx) - it is almost ready to ship. 
```

Comment:
again, like above, dont draw attention to just work. 

in this particular case, there's something interesting to note, for example:
- @RichardLitt improved the `gx` README document to better explain its goals, and help people get started. Take a look and give us feedback through an issue if you think anything could be improved.